### DISCUSSION TITLE
Popover 里弹出 Modal, 跑到 Popover 后面去了

### DISCUSSION THREAD

**Discussion by ekuku1984** [ant-design/ant-design#40518] [General]
感觉这些弹层组件的 z-index 应该要统一管理，不要各种组件有自己的 z-index, 导致前后关系不好确定，大家觉得呢？

**Comment by yoyo837** [comment]
搞一个PopopIndexManager，可以取next value？

**Comment by SignDawn** [comment]
props 里不是有 zIndex 属性？

**Reply by ekuku1984** [reply]
如果自己设置 zIndex 属性， 这个显示太麻烦了

**Reply by SignDawn** [reply]
如果你需要在 Modal 里展示 Popover 呢？
